be/src/olap/types.h

```diff
@@ -255,6 +255,19 @@ struct BaseFieldtypeTraits : public CppTypeTraits<field_type> {
     }
 };
 
+#define CONVERT_FROM_VARCHAR(type_name, string_parser_fn) \
```

Comment:
Can you implement this in a template function other than a macro?
Using macro is error-prone, we should avoid this as far as we can